backend/Functions/Edna.AssignmentLinks/Edna.AssignmentLinks/AssignmentLinksApi.cs

```diff
@@ -109,6 +109,16 @@ public AssignmentLinksApi(ILogger<AssignmentLinksApi> logger, IMapper mapper)
             if (linkId != linkDto.Id)
                 return new BadRequestErrorMessageResult("The provided link content doesn't match the path.");
 
+            Uri linkUri;
```

Comment:
I'd recommend using DataAnnotations for C#. It would be much cleaner and scalable for future.

> **Reference**: https://docs.microsoft.com/en-us/aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-validation-to-the-model

Reply:
Thank you for this suggestion. I have made the necessary changes to use data annotations and validation.